jupyter-book/introduction/interoperability.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<p>I don't know in how much detail you want to mention this here, but there are some issues with the conversion from the latest anndata format: seurat disk does not work with <code>.layers</code> and still checks <code>.raw</code> (see https://mojaveazure.github.io/seurat-disk/reference/Convert.html). Also for the neighbour's graph it looks at <code>/uns/neighbors/distances</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Also just learnt that h5seurat doesn't work with the latest Seurat version https://github.com/mojaveazure/seurat-disk/issues/154</p>

Reply:
<p>Hmmm...not entirely sure what to do here. This example works and I'm not sure we can mention every potential issues. The other packages also have issues with specific cases.</p>

Reply:
We can consider adding one sentence on potential limitations but ultimately point to their docs. We can't keep this in sync anyways.